src/main/java/ch/kalunight/zoe/repositories/ServerRepository.java

```diff
@@ -0,0 +1,156 @@
+package ch.kalunight.zoe.repositories;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import ch.kalunight.zoe.model.dto.DTO;
+
+public class ServerRepository {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'ServerRepository' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=4739924)